moskito-web/src/main/java/net/anotheria/moskito/web/filters/JourneyFilter.java

```diff
@@ -93,7 +96,18 @@
 		}
 
 		//set custom tags
-
+		for (CustomTag tag : taggingConfig.getCustomTags()) {
+			if (TagPrefix.HEADER.getName().equals(tag.getPrefix()) && !StringUtils.isEmpty(req.getHeader(tag.getAttributeName()))) {
+				MoSKitoContext.addTag(tag.getName(), req.getHeader(tag.getAttributeName()));
+			} else if (TagPrefix.REQUEST.getName().equals(tag.getPrefix()) && req.getAttribute(tag.getAttributeName()) != null) {
+				MoSKitoContext.addTag(tag.getName(), String.valueOf(req.getAttribute(tag.getAttributeName())));
+			} else if (TagPrefix.SESSION.getName().equals(tag.getPrefix()) && req.getSession().getAttribute(tag.getAttributeName()) != null) {
+				HttpSession session = req.getSession();
```

Comment:
just in case, but please make req.getSession(false) and check for null. Otherwise moskito will create a log of http sessions in stateless apps. (Already happened).